classifiers/llm/bert_toxicity_classifier/config.py

```diff
@@ -0,0 +1,46 @@
+from util.configs import build_classifier_premium_config
+from util.enums import State, SelectionType, BricksVariableType, RefineryDataType
+from . import bert_toxicity_classifier, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_classifier_premium_config(
+        function=bert_toxicity_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=80,
+        tabler_icon="MoodAngry",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        gdpr_compliant="true",
```

Comment:
this true? Is huggingface generally GDPR compliant?

Reply:
Probably not! Will change.